src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/VisitService.kt

```diff
@@ -100,7 +97,9 @@ class VisitService(
     val application = applicationService.getApplicationEntity(applicationReference)
 
     val existingBooking = visitRepository.findVisitByApplicationReference(application.reference)
-    checkSlotCapacity(bookingRequestDto, application, existingBooking)
+
+    // application validity checks
+    applicationValidationService.isApplicationValid(bookingRequestDto, application, existingBooking)
```

Comment:
We also need a call from the application service right?

Reply:
to be included in https://dsdmoj.atlassian.net/browse/VB-4085